rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    @book.rentals << self
+    @person.rentals << self
+  end
+end
```

Comment:
- [ ] Kindly let's add the setters and getters for the @date instance variable: `attr_accessor :date`.
- [ ]  Getters and Setters for @book and @person instance variables should be present as well: ` attr_accessor :date, :book, :person`

Reply:
it's done.